app/views/display_tasks/_header.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="header">
```

Comment:
Hmmm - I think you should be able to put this view into a shared folder so that all of the different areas can use it and you wouldn't need to have this file in your application more than once